ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -43,4 +43,7 @@ ssh "${SSHOPTS[@]}" "root@${IP}" tar -czf - /tmp/artifacts | tar -C "${ARTIFACT_
 
 set -e
 echo "### Done! (${rv})"
+if [ $rv -eq 0 ]; then
+    touch $SHARED_DIR/e2e_test_complete
```

Comment:
```suggestion
    touch "${SHARED_DIR}/e2e_test_complete"
```

I'm not sure why shellcheck didn't complain, but this should be quoted